packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -417,6 +420,17 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// {@endtemplate}
   final bool transitionBetweenRoutes;
 
+  /// {@template flutter.cupertino.CupertinoNavigationBar.enableBackgroundFilterBlur}
+  /// Whether to have blur effect with non opaque color.
+  ///
+  /// When [enableBackgroundFilterBlur] is set to false, the blur effect will be
+  /// disabled. [enableBackgroundFilterBlur]'s behaviour will only be respected when
```

Comment:
```suggestion
  /// disabled. The behavior of [enableBackgroundFilterBlur] will only be respected when
```
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar